spec/features/user_visits_place_spec.rb

```diff
@@ -14,6 +14,24 @@
         and_return(Place.first)
     end
 
+    xit "with no previous entry" do
+      VCR.use_cassette("existing_place") do
+        visit "/"
+
+        page.fill_in 'nav-search',
+          :with => 'Turing School of Software & Design'
+
+        click_button "Go"
+
+        assert_equal '/places/turing-school-of-software-design',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.